packages/aws-helpers/src/stagedRollouts/functions/configOriginRequest.ts

```diff
@@ -0,0 +1,68 @@
+import {
+    CloudFrontRequestEvent,
+    CloudFrontResponse,
+    defineLambdaEdgeRequestHandler,
+    setHeader
+} from "~/lambdaEdge";
+
+import { get } from "https";
+
+export default defineLambdaEdgeRequestHandler(async event => {
+    try {
+        const configJson = await loadConfigJson(event);
+        const response: CloudFrontResponse = {
+            status: "200",
+            statusDescription: "OK",
+            headers: {},
+            body: configJson
+        };
+
+        setHeader(response.headers, {
```

Comment:
Just a note - looking at this file, it seems to us that this is the old code. For example the `setHeader` is already renamed to `setResponseHeader` in the previous PR.

So, there's gonna be some merging most probably.

Reply:
You can change headers for both response and request and they have the same structure in `Lambda@Edge` so I used a more generic approach, so that you can use the same function for request and response headers.